bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-participants/user-list-participants/list-item/component.tsx

```diff
@@ -240,8 +246,12 @@ const UserListItem: React.FC<UserListItemProps> = ({ user, lockSettings, index }
         talking={voiceUser?.talking}
         muted={voiceUser?.muted}
         listenOnly={voiceUser?.listenOnly || voiceUser?.listenOnlyInputDevice}
-        voice={voiceUser?.joined}
-        noVoice={!voiceUser?.joined}
+        voice={isLiveKit
+          ? (voiceUser?.joined && !voiceUser?.deafened)
+          : voiceUser?.joined}
+        noVoice={isLiveKit
+          ? (voiceUser?.joined && voiceUser?.deafened)
+          : !voiceUser?.joined}
```

Comment:
I think this doesn't need to be LiveKit specific.
Other bridge users should also be flagged as "not in audio" if they're deafened - even though the deafen primitive is currently not used for them.
Having a generic check here makes this future-proof and simpler.
(See previous review comment)

```suggestion
        voice={voiceUser?.joined && !voiceUser?.deafened}
        noVoice={!voiceUser?.joined || voiceUser?.deafened}
```